admin/lib/solidus_admin/configuration.rb

```diff
@@ -38,6 +38,15 @@ class Configuration < Spree::Preferences::Configuration
     #
     # Recompile with `bin/rails solidus_admin:tailwindcss:build` after changing this list.
     preference :tailwind_stylesheets, :array, default: []
+
+    preference :importmap_cache_sweepers, :array, default: [
```

Comment:
It'd be nice if we document this preference and also add it to the [generated initializer](https://github.com/solidusio/solidus/blob/nebulab/admin/admin/lib/generators/solidus_admin/install/templates/config/initializers/solidus_admin.rb).

Do you think it would make sense to add `Rails.root.join("app", "javascript", "solidus_admin")` & `Rails.root.join("app", "assets", "javascripts", "solidus_admin")` to the defaults?